src/test/kotlin/no/nav/syfo/testhelper/generator/FellesformatGenerator.kt

```diff
@@ -0,0 +1,104 @@
+package no.nav.syfo.testhelper.generator
+
+fun generateFellesformatMessage(): String {
+    return "<\\?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"\\?>\n" +
+        "<EI_fellesformat xmlns=\"http://www.nav.no/xml/eiff/2/\" xmlns:ns6=\"http://www.kith.no/xmlstds/base64container\" xmlns:ns5=\"http://www.kith.no/xmlstds/felleskomponent1\" xmlns:ns2=\"http://www.kith.no/xmlstds/msghead/2006-05-24\" xmlns:ns4=\"http://www.kith.no/xmlstds/dialog/2006-10-11\" xmlns:ns3=\"http://www.w3.org/2000/09/xmldsig#\">\n" +
+        "    <ns2:MsgHead>\n" +
+        "        <ns2:MsgInfo>\n" +
+        "            <ns2:Type V=\"DIALOG_NOTAT\" DN=\"Notat\"/>\n" +
+        "            <ns2:MIGversion>v1.2 2006-05-24</ns2:MIGversion>\n" +
+        "            <ns2:GenDate>\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{0,7}</ns2:GenDate>\n" +
+        "            <ns2:MsgId>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}</ns2:MsgId>\n" +
```

Comment:
Det er kanskje ikke så lett å se hvor Regex kommer inn i bildet?
Jeg vurderte å gjøre om til dette:
```
<ns2:GenDate>${Regex("\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{0,7}")}</ns2:GenDate>\n" +
```
For at det skulle være tydelig hvor man bruker regex, men det er kanskje bare mer forvirrende, siden man nå lager Regex som blir til String, som så blir til Regex igjen i testen? Noen som har tanker om dette?